addons/mail/static/src/core/common/notification_message.js

```diff
@@ -51,14 +51,17 @@ export class NotificationMessage extends Component {
     }
 
     get callInformation() {
-        if (this.message.create_date.equals(this.message.write_date)) {
+        let duration = this.message.call_history_ids[0]?.duration
+            ? luxon.Duration.fromObject({
+                  seconds: Math.max(
+                      1,
+                      Math.round(this.message.call_history_ids[0].duration * 3600)
+                  ),
+              }).shiftTo("hours", "minutes", "seconds")
+            : null;
+        if (!duration) {
```

Comment:
This should be simpler: 
```suggestion
        if (!this.message.call_history_ids[0]?.duration) {
            return _t("%(author)s started a call.", { author: this.message.author.name });
        }
        let duration = luxon.Duration.fromObject({
                  seconds: Math.max(
                      1,
                      Math.round(this.message.call_history_ids[0].duration * 3600)
                  ),
              }).shiftTo("hours", "minutes", "seconds")
```